Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -323,4 +246,13 @@ class ThumbnailCell: UICollectionViewCell {
             }
         }
     }
+
+    func extractImageFromCell() -> UIImage? {
```

Comment:
@bkmunar, yes the ideal approach would have been to just return the imageView.image and UIColor references. But, there are 2 scenarios for ThumbnailCells: TopSites and default Suggested TopSites. For the former case, the UIColor for background reference doesn't exist. It is a UIImage for background. And, for the latter case, the backgroundImage property is not used. Instead, a UIColor is applied directly to the UI's contentView. The imagePadding is also not applied to this case.

So, instead of duplicating the scenario specific logic, or encapsulating into a function that could any turn out to not be necessary if the TopSitePanel goes away, I figured I leave it at this.

@farhanpatel, any suggestions on handling this? The main side-effect here is described in [this comment](https://github.com/mozilla-mobile/firefox-ios/pull/2268#discussion_r94798133)